textstat/textstat.py

```diff
@@ -398,6 +397,30 @@ def text_standard(self, text, float_output=None):
                 lower_score, get_grade_suffix(lower_score),
                 upper_score, get_grade_suffix(upper_score)
             )
+    @repoze.lru.lru_cache(maxsize=128)
+    def spache_readability(self, text, float_output=False):
+        """
+        Function to calculate SPACHE readability formula for young readers.
+        I/P - a text
+        O/P - an int Spache Readability Index/Grade Level
+        """
+        #Count the total number of words in the sample text
+        total_no_of_words = self.lexicon_count(text)
+        #Count the number of sentences in the sample text
+        count_of_sentences = self.sentence_count(text)
+        #find the average sentance length
+        asl = total_no_of_words/count_of_sentences
+        text_set = set(self.remove_punctuation(text).split())
```

Comment:
There is a function in textstat `difficult_words()`, would this be suitable for calculating the percentage here?
```python
pdw = (self.difficult_words(text)/total_no_of_words) * 100
```